decidim-blogs/lib/decidim/api/post_type.rb

```diff
@@ -7,6 +7,7 @@ class PostType < Decidim::Api::Types::BaseObject
       implements Decidim::Comments::CommentableInterface
       implements Decidim::Core::AttachableInterface
       implements Decidim::Core::AuthorableInterface
+      implements Decidim::Core::FollowableInterface
```

Comment:
Let's add the new field (`followsCount`) to `decidim-blogs/spec/types/integration_schema_spec.rb`

Reply:
I am assuming you already reviewed everything from this PR, so i have implement it in #14358 , commit: 0609b872aca75ba4de3a97eb0834b4cbcc58dd48